docs/source/info_gain.rst

```diff
@@ -0,0 +1,227 @@
+
+    import matplotlib.pyplot as plt
+
+    def create_trajectory(
+        energy_landscape: Callable[[float, float], float], file_name: str
+    ) -> np.ndarray:
+        """Simulate Langevin Dynamics on a given energy landscape."""
+        rng = np.random.default_rng(0)
+        n_atoms = 100
+        time_steps = 10000
+        dt = 0.01  # Time step
+        diffusion_coeff = 0.6  # Diffusion coefficient (random noise strength)
+
+        if energy_landscape == energy_landscape_1:
+            particles = rng.standard_normal((n_atoms, 2)) * 0.2
+            particles[n_atoms // 2 :, 1] += 1.0
+        else:
+            particles = rng.standard_normal((n_atoms, 2)) * 0.2
+            n_group = n_atoms // 4
+            particles[n_group : 2 * n_group, 1] += 1  # (0, 1)
+            particles[2 * n_group : 3 * n_group, 0] += 1  # (1, 0)
+            particles[3 * n_group :, 0] += 1  # (1, 1)
+            particles[3 * n_group :, 1] += 1
+
+        trajectory = np.zeros((time_steps, n_atoms, 2))
+        for t in range(time_steps):
+            for i in range(n_atoms):
+                x, y = particles[i]
+                fx, fy = numerical_gradient(energy_landscape, x, y)
+                noise_x = np.sqrt(2 * diffusion_coeff * dt) * rng.standard_normal()
+                noise_y = np.sqrt(2 * diffusion_coeff * dt) * rng.standard_normal()
+
+                # Update position with deterministic force and stochastic term
+                particles[i, 0] += fx * dt + noise_x
+                particles[i, 1] += fy * dt + noise_y
+
+                trajectory[t, i] = particles[i]
+
+        plt.figure()
+        plt.plot(trajectory[:, :, 0], trajectory[:, :, 1])
+        plt.show()
+
+        dataset = np.transpose(trajectory, (1, 0, 2))
+        np.save(f"info_gain/{file_name}.npy", dataset)
+        return dataset
+
+
+Let's simulate the trajectories and store them in two variables. We also save them as .npy files so that we don't have to simulate them every time. 
+
+.. code-block:: python
+
+    from pathlib import Path
+
+    file_1 = Path("info_gain/trj_2.npy")  #  With 2 minima
+    file_2 = Path("info_gain/trj_4.npy")  #  With 4 minima
+    if file_1.exists():
+        dataset_1 = np.load(file_1)
+    else:
+        dataset_1 = create_trajectory(energy_landscape_1, "trj_2")
+    if file_2.exists():
+        dataset_2 = np.load(file_2)
+    else:
+        dataset_2 = create_trajectory(energy_landscape_2, "trj_4")
+
+
+Now that we have the trajectories, we can, for each one, perform Onion clustering and computing the corresponding information gain, using either 
+
+- only the y coordinate, or
+- both x and y coordinates
+
+For each case, we do the analysis for a range of values of the Onion clustering time resolution ∆t. The information gain for each analysis is saved in the variable "results". 
+To check if the clustering is working in a meaningful way, we also plot the results of Onion clustering for one specific value of ∆t. 
```

Comment:
This would be a good place for the plot, maybe annotated to discuss how you know its good?